consensus/istanbul/backend/backend.go

```diff
@@ -867,6 +881,15 @@ func (sb *Backend) retrieveValidatorConnSet() (map[common.Address]bool, error) {
 	return sb.cachedValidatorConnSet, nil
 }
 
+// retrieveCachedValidatorConnSet returns the most recently cached validator conn
```

Comment:
Please update this comment to be accurate. Also, I noticed that the only way this cache is updated now is a call to `updateCachedValidatorConnSet`. I think this is fine, but I wanted to make sure it is intentional.